### PR TITLE
add ember octane / native class example to readme

### PR DESCRIPTION
**Pull request by ballPointPenguin**
After some trial and error, I found this pattern to work well for me, in a few new Octane apps that I have.
I am open to suggestions or revisions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,6 +235,55 @@ set(this, 'metrics.context.userName', 'Jimbo');
...
+import { inject as service } from '@ember/service'
+import { scheduleOnce } from '@ember/runloop'
+
+export default class ApplicationRoute extends Route {
+  @service metrics
+  @service router
+
+  @action
+  didTransition() {
+    this._super(...arguments)
```
Comment (lifeart) [inline README.md]: should it be 
```js
didTransition() {
     super.didTransition(...arguments);
}

```

?

Comment (ballPointPenguin) [reply README.md]: I think you're right. I'll test it to be sure.

File: README.md [hunk 2]
```diff
@@ -235,6 +235,55 @@ set(this, 'metrics.context.userName', 'Jimbo');
...
+    this._trackPage()
+  }
+
+  _trackPage = function() {
```
Comment (NullVoxPopuli) [inline README.md]: I think this should be:
```ts
_trackPage() {
 // ...
}
```

File: README.md [hunk 3]
```diff
@@ -235,6 +235,55 @@ set(this, 'metrics.context.userName', 'Jimbo');
...
+
+```js
+// app/routes/home.js
+import Route from '@ember/routing/route'
+import { action } from '@ember/object'
+
+export default class HomeRoute extends Route {
+  @action
+  didTransition() {
+    this._super(...arguments)
```
Comment (NullVoxPopuli) [inline README.md]: super.didTransition here too :)

**Comment by ballPointPenguin** [comment]
Thanks for awesome feedback. This improves my own understanding of Native Class use in Ember and more! I've been busy but will  update this very soon with improvements.

File: README.md (hunk 1 above)
Comment (ballPointPenguin) [reply README.md]: changing
`this._super(...arguments);`
to
`super.didTransition(...arguments);`

yields this error:
```
TypeError: (intermediate value).didTransition is not a function
    at ApplicationRoute.didTransition (/tmp/broccoli-28387PLcPVSpmiYuJ/out-275-broccoli_merge_trees/assets/aliencyborg/routes/application.js:28:1)
    at Class.triggerEvent (/tmp/broccoli-28387PLcPVSpmiYuJ/out-275-broccoli_merge_trees/assets/@ember/-internals/routing/lib/system/router.js:1290:1)
...
```

File: README.md (hunk 2 above)
Comment (ballPointPenguin) [reply README.md]: yes

File: README.md (hunk 3 above)
Comment (ballPointPenguin) [reply README.md]: same error as above

**Comment by ballPointPenguin** [comment]
IMO this is correct and good to go.

**Pull request merged**